pkg/config/config.go

```diff
@@ -0,0 +1,224 @@
+package config
```

Comment:
Lots of boiler plate in this file, perhaps there is a library out there already supporting this? 

https://github.com/spf13/viper maybe?

Reply:
Hmmm. I wonder about merging this and making that a separate issue/PR.